Fix for PSP/Android and added new sets M20/MH1/SS2/C19 and fixed UST/V17 sets

Vitty85 opened the pull request:
Implemented the Image downloader for Android and corrected several build issues for PSP and Android.

Replaced tabs by 4 spaces.

xawotihs commented:
- See cross-compilation errors coming from Travis.
- I'm not fan of your modification of boost. I did a lot of work in the cmake branch to remove the usage of boost, I'll try to backport part of it in the master branch in the following days.
- I noticed you increased a lot the version of the Android SDK Wagic needs, why did you increase it that much ?

Vitty85 commented:
Hi,

I increased sdk version because 10 is no longer supported. I used the last available sdk, not mandatory, I had that one installed.

I had to slightly modify boost in order to compile for psp and Android, if you can find different solution for me no problem.

I will take a look to compile error ASAP.

See you later.

--
Inviato da iPhone

> Il giorno 7 set 2019, alle ore 13:03, Xawotihs <notifications@github.com> ha scritto:
> 
> See cross-compilation errors coming from Travis.
> I'm not fan of your modification of boost. I did a lot of work in the cmake branch to remove the usage of boost, I'll try to backport part of it in the master branch in the following days.
> I noticed you increased a lot the version of the Android SDK Wagic needs, why did you increase it that much ?
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.

Vitty85 commented:
Hi,

In short I will commit fix for Travis error and some fix for snow mana cost.

Hope you can easily merge.

--
Inviato da iPhone

> Il giorno 7 set 2019, alle ore 13:03, Xawotihs <notifications@github.com> ha scritto:
> 
> See cross-compilation errors coming from Travis.
> I'm not fan of your modification of boost. I did a lot of work in the cmake branch to remove the usage of boost, I'll try to backport part of it in the master branch in the following days.
> I noticed you increased a lot the version of the Android SDK Wagic needs, why did you increase it that much ?
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.

Vitty85 commented:
Hi,

I commit my modifications, i reverted boost, and some error disappears.
Just Travis CI gave errors on Java:

/home/travis/build/WagicProject/wagic/projects/mtg/Android/src/net/wagic/utils/ImgDownloader.java:19:
error: cannot find symbol
    [javac] import java.nio.charset.StandardCharsets;

But with my SDK i don't have any problem to include java.nio package.

Do you have any ideas why on toolchain it gives this error?

Il giorno sab 7 set 2019 alle ore 19:04 Vittorio Alfieri <
vittorioalfieri@gmail.com> ha scritto:

> Hi,
>
> In short I will commit fix for Travis error and some fix for snow mana
> cost.
>
> Hope you can easily merge.
>
> --
> Inviato da iPhone
>
> Il giorno 7 set 2019, alle ore 13:03, Xawotihs <notifications@github.com>
> ha scritto:
>
>
>    - See cross-compilation errors coming from Travis.
>    - I'm not fan of your modification of boost. I did a lot of work in
>    the cmake branch to remove the usage of boost, I'll try to backport part of
>    it in the master branch in the following days.
>    - I noticed you increased a lot the version of the Android SDK Wagic
>    needs, why did you increase it that much ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/WagicProject/wagic/pull/1024?email_source=notifications&email_token=AMVK66AE5EDI3AEZSUL2VBDQIODA3A5CNFSM4IUFQC3KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD6EWKUQ#issuecomment-529098066>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMVK66ERASWKZX7KZUSV4I3QIODA3ANCNFSM4IUFQC3A>
> .
>
>

-- 
*Ing. Vittorio Alfieri*

Vitty85 commented:
I understood,

The Toolchain is using too old API for Android. The java.nio packages needs
API level 26 or higher.

Do you think it's possible to update SDK on Toolchain? My Image downloader
for Android needs that package to work properly.

Let me know, see you soon!

Il giorno sab 7 set 2019 alle ore 23:33 Vittorio Alfieri <
vittorioalfieri@gmail.com> ha scritto:

> Hi,
>
> I commit my modifications, i reverted boost, and some error disappears.
> Just Travis CI gave errors on Java:
>
> /home/travis/build/WagicProject/wagic/projects/mtg/Android/src/net/wagic/utils/ImgDownloader.java:19:
> error: cannot find symbol
>     [javac] import java.nio.charset.StandardCharsets;
>
> But with my SDK i don't have any problem to include java.nio package.
>
> Do you have any ideas why on toolchain it gives this error?
>
> Il giorno sab 7 set 2019 alle ore 19:04 Vittorio Alfieri <
> vittorioalfieri@gmail.com> ha scritto:
>
>> Hi,
>>
>> In short I will commit fix for Travis error and some fix for snow mana
>> cost.
>>
>> Hope you can easily merge.
>>
>> --
>> Inviato da iPhone
>>
>> Il giorno 7 set 2019, alle ore 13:03, Xawotihs <notifications@github.com>
>> ha scritto:
>>
>>
>>    - See cross-compilation errors coming from Travis.
>>    - I'm not fan of your modification of boost. I did a lot of work in
>>    the cmake branch to remove the usage of boost, I'll try to backport part of
>>    it in the master branch in the following days.
>>    - I noticed you increased a lot the version of the Android SDK Wagic
>>    needs, why did you increase it that much ?
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/WagicProject/wagic/pull/1024?email_source=notifications&email_token=AMVK66AE5EDI3AEZSUL2VBDQIODA3A5CNFSM4IUFQC3KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD6EWKUQ#issuecomment-529098066>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AMVK66ERASWKZX7KZUSV4I3QIODA3ANCNFSM4IUFQC3A>
>> .
>>
>>
>
> --
> *Ing. Vittorio Alfieri*
>


-- 
*Ing. Vittorio Alfieri*

Vitty85 commented:
Ok,

I refactored code to remove java.nio dependencies, i committed and now i'm
waiting to see if everything is ok.

See you.

Il giorno sab 7 set 2019 alle ore 23:39 Vittorio Alfieri <
vittorioalfieri@gmail.com> ha scritto:

> I understood,
>
> The Toolchain is using too old API for Android. The java.nio packages
> needs API level 26 or higher.
>
> Do you think it's possible to update SDK on Toolchain? My Image downloader
> for Android needs that package to work properly.
>
> Let me know, see you soon!
>
> Il giorno sab 7 set 2019 alle ore 23:33 Vittorio Alfieri <
> vittorioalfieri@gmail.com> ha scritto:
>
>> Hi,
>>
>> I commit my modifications, i reverted boost, and some error disappears.
>> Just Travis CI gave errors on Java:
>>
>> /home/travis/build/WagicProject/wagic/projects/mtg/Android/src/net/wagic/utils/ImgDownloader.java:19:
>> error: cannot find symbol
>>     [javac] import java.nio.charset.StandardCharsets;
>>
>> But with my SDK i don't have any problem to include java.nio package.
>>
>> Do you have any ideas why on toolchain it gives this error?
>>
>> Il giorno sab 7 set 2019 alle ore 19:04 Vittorio Alfieri <
>> vittorioalfieri@gmail.com> ha scritto:
>>
>>> Hi,
>>>
>>> In short I will commit fix for Travis error and some fix for snow mana
>>> cost.
>>>
>>> Hope you can easily merge.
>>>
>>> --
>>> Inviato da iPhone
>>>
>>> Il giorno 7 set 2019, alle ore 13:03, Xawotihs <notifications@github.com>
>>> ha scritto:
>>>
>>>
>>>    - See cross-compilation errors coming from Travis.
>>>    - I'm not fan of your modification of boost. I did a lot of work in
>>>    the cmake branch to remove the usage of boost, I'll try to backport part of
>>>    it in the master branch in the following days.
>>>    - I noticed you increased a lot the version of the Android SDK Wagic
>>>    needs, why did you increase it that much ?
>>>
>>> —
>>> You are receiving this because you authored the thread.
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/WagicProject/wagic/pull/1024?email_source=notifications&email_token=AMVK66AE5EDI3AEZSUL2VBDQIODA3A5CNFSM4IUFQC3KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD6EWKUQ#issuecomment-529098066>,
>>> or mute the thread
>>> <https://github.com/notifications/unsubscribe-auth/AMVK66ERASWKZX7KZUSV4I3QIODA3ANCNFSM4IUFQC3A>
>>> .
>>>
>>>
>>
>> --
>> *Ing. Vittorio Alfieri*
>>
>
>
> --
> *Ing. Vittorio Alfieri*
>


-- 
*Ing. Vittorio Alfieri*

Vitty85 commented:
Hi,

I finally solved all the compilation error for both C++ and Java, but the
test suite return 2 errors at the end. Do yuo have any idea?

We are near to merge i guess, let me know when you can find time!

Il giorno dom 8 set 2019 alle ore 10:00 Vittorio Alfieri <
vittorioalfieri@gmail.com> ha scritto:

> Ok,
>
> I refactored code to remove java.nio dependencies, i committed and now i'm
> waiting to see if everything is ok.
>
> See you.
>
> Il giorno sab 7 set 2019 alle ore 23:39 Vittorio Alfieri <
> vittorioalfieri@gmail.com> ha scritto:
>
>> I understood,
>>
>> The Toolchain is using too old API for Android. The java.nio packages
>> needs API level 26 or higher.
>>
>> Do you think it's possible to update SDK on Toolchain? My Image
>> downloader for Android needs that package to work properly.
>>
>> Let me know, see you soon!
>>
>> Il giorno sab 7 set 2019 alle ore 23:33 Vittorio Alfieri <
>> vittorioalfieri@gmail.com> ha scritto:
>>
>>> Hi,
>>>
>>> I commit my modifications, i reverted boost, and some error disappears.
>>> Just Travis CI gave errors on Java:
>>>
>>> /home/travis/build/WagicProject/wagic/projects/mtg/Android/src/net/wagic/utils/ImgDownloader.java:19:
>>> error: cannot find symbol
>>>     [javac] import java.nio.charset.StandardCharsets;
>>>
>>> But with my SDK i don't have any problem to include java.nio package.
>>>
>>> Do you have any ideas why on toolchain it gives this error?
>>>
>>> Il giorno sab 7 set 2019 alle ore 19:04 Vittorio Alfieri <
>>> vittorioalfieri@gmail.com> ha scritto:
>>>
>>>> Hi,
>>>>
>>>> In short I will commit fix for Travis error and some fix for snow mana
>>>> cost.
>>>>
>>>> Hope you can easily merge.
>>>>
>>>> --
>>>> Inviato da iPhone
>>>>
>>>> Il giorno 7 set 2019, alle ore 13:03, Xawotihs <
>>>> notifications@github.com> ha scritto:
>>>>
>>>>
>>>>    - See cross-compilation errors coming from Travis.
>>>>    - I'm not fan of your modification of boost. I did a lot of work in
>>>>    the cmake branch to remove the usage of boost, I'll try to backport part of
>>>>    it in the master branch in the following days.
>>>>    - I noticed you increased a lot the version of the Android SDK
>>>>    Wagic needs, why did you increase it that much ?
>>>>
>>>> —
>>>> You are receiving this because you authored the thread.
>>>> Reply to this email directly, view it on GitHub
>>>> <https://github.com/WagicProject/wagic/pull/1024?email_source=notifications&email_token=AMVK66AE5EDI3AEZSUL2VBDQIODA3A5CNFSM4IUFQC3KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD6EWKUQ#issuecomment-529098066>,
>>>> or mute the thread
>>>> <https://github.com/notifications/unsubscribe-auth/AMVK66ERASWKZX7KZUSV4I3QIODA3ANCNFSM4IUFQC3A>
>>>> .
>>>>
>>>>
>>>
>>> --
>>> *Ing. Vittorio Alfieri*
>>>
>>
>>
>> --
>> *Ing. Vittorio Alfieri*
>>
>
>
> --
> *Ing. Vittorio Alfieri*
>


-- 
*Ing. Vittorio Alfieri*
